src/main/java/Server/MopeLSPServerLauncher.java

```diff
@@ -126,11 +134,13 @@ public static void readConfig() {
             configPath = Path.of(home,"\\mope\\server.conf");
             try{
                 readConfigFile(configPath);
-            } catch (Exception ex){
+            } catch (IOException ex){
                 configPath = Path.of("src/main/java/Server/server.config");
                 try {
                     readConfigFile(configPath);
-                } catch (Exception exc) {}
+                } catch (IOException exc) {
+                    throw new AssertionError("fatal error, could not read config file");
```

Comment:
You should propagate `exc` by using it as a second argument. Also, you can tell the user where they should *put* the configuration file, so that they can maybe fix this error.